components/Onboarding/ConnectViaWallet/useInitConnectViaWalletState.ts

```diff
@@ -10,12 +10,16 @@ import {
 } from "thirdweb/react";
 
 import { useConnectViaWalletContext } from "./ConnectViaWallet.context";
+import { ensureError } from "../../../utils/error";
 import { thirdwebClient } from "../../../utils/thirdweb";
 import { isOnXmtp } from "../../../utils/xmtpRN/client";
 import { getInboxId } from "../../../utils/xmtpRN/signIn";
 
-// For now let's keep Thirdweb and the hooks because I haven't found a better way to do it.
-// But ideally a lot of this it outside React.
+/**
+ * For now let's keep Thirdweb and the hooks because I haven't found a better way to do it.
```

Comment:
Yeah there's probably some way to hook into these without react, but not sure how much is being exposed.

It's using react-query under the hood, but the query client is not exposed
Maybe some way to patch package it if we really need to